src/pages/ThorChainLP/components/ReusableLpStatus/TransactionRow.tsx

```diff
@@ -179,6 +191,8 @@ export const TransactionRow: React.FC<TransactionRowProps> = ({
       if (isRuneTx && opportunityType !== 'sym')
         poolAssetId = assetIdToPoolAssetId({ assetId: thorchainAssetId })
       if (!isRuneTx && opportunityType !== 'sym') poolAssetId = assetIdToPoolAssetId({ assetId })
+      if (opportunityType === AsymSide.Asset && type === 'sym')
```

Comment:
If the manually selected withdraw type is asym `asset` but the current opportunity type is `sym`, we should use `RUNE` as fee asset and send a tx using thorchain, so the pool asset id is the asset but the TX will be using RUNE

Reply:
Do we want to add that comment as a comment?

Reply:
Well I don't think so, or we would comment all these conditions